### PR TITLE
Double click to freeze tooltip

### PR DESCRIPTION
**Pull request by zackhsi**
Typing is painful, and there isn't an easy way to copy the series names.

screen shot 2015-11-17 at 4 43 56 pm

### CODE REVIEW

**Comment by zackhsi** [comment]
Wanted to also grab the values, but that's nontrivial.

**Comment by zackhsi** [comment]
@danielhochman

File: public/app/panels/graph/graph.tooltip.js [hunk 1]
```diff
@@ -86,6 +86,20 @@ function ($) {
       return results;
     };
 
+    elem.dblclick(function () {
+      var seriesList = getSeriesFn();
+      var title = "Targets:\n\n";
+      var message = "";
+      for (var i = 0; i < seriesList.length; i++) {
```
Comment (rowillia) [inline public/app/panels/graph/graph.tooltip.js]: If you want to get super fucking clever:

``` js
message = seriesList.map(function(x) { 
  return x.alias; 
}.join('\n\n');
```

**Comment by rowillia** [comment]
+1

**Comment by theatrus** [comment]
:heart_eyes_cat:

**Comment by paulnivin** [comment]
100000000000000000000000000x :+1:

**Comment by zackhsi** [comment]
:rocket:

**Comment by lyft-buildnotify** [comment]
Submit queue bot says: Your commit has been enqueued for commit to master followed by a test run. I will close and update this PR once tests have passed and your code is ready for deployment...

**Comment by mattklein123** [comment]
commit of the week, no year

**Pull request merged**